### PR TITLE
Wasm Smart Contract Bounty (Ink!) grant application

### PR DESCRIPTION
**Pull request by Burmenska**
We would like to develop a Wasm Smart Contract Bounty (Ink!) project.

### CODE REVIEW

**Comment by TtomaS7** [comment]
Hello @Burmenska 
Thank you for your application.

Upon our initial review, we noticed that the information provided seems to be a direct copy of the provided template (deliverables etc.). As per our requirements, we need specific details about your application.
Additionally, It appears that you've included an Ethereum address, whereas our requirement is for a Polkadot address. 
Please add relevant information to the application, or I need to close this PR. Thank you

**Review by TtomaS7 — CHANGES_REQUESTED** [review]
please provide more details.

File: applications/Wasm Smart Contract Bounty (Ink!).md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+        value: Balance,
+        deadline: u128,
+        v: u8,
+        r: Hash,
+        s: Hash,
+    ) -> Result<()>;
+}
+​
+#[ink::trait_definition]
+pub trait InkswapERC20: InkswapERC20 {
```
Comment (mike1729) [inline applications/Wasm Smart Contract Bounty (Ink!).md:86]: Some typo.

**Review by mike1729 — CHANGES_REQUESTED** [review]
Bare interfaces are not enough to understand what you actually plan to implement. Please provided overview and detailed description.

File: applications/Wasm Smart Contract Bounty (Ink!).md (hunk 1 above)
Comment (TtomaS7) [reply applications/Wasm Smart Contract Bounty (Ink!).md:86]: yes, agree with Michael 
Please take a look on this grant application - https://github.com/Chidasan/Ecosystem-Grants/blob/3be440fbb272358a4c7d3d0ee0539f4529bb0961/applications/starlay_finance.md
or others

**Review by HCastano — CHANGES_REQUESTED** [review]
I'm not sold on this application. The vision for how this will positively impact the ink! and Polkadot ecosystem isn't there for me.

I also don't see any Rust repositories on the CIDT GitHub, which also doesn't give me that much confidence.

I'd recommend you go out and build some ink! contracts and apply again in the future, using those as showcases for what you can do.

**Comment by TtomaS7** [comment]
I would like to start off by expressing our deepest appreciation for the time, effort, and innovation you have put into your application for our Ecosystem Grant.

Regrettably, after a careful and thorough review, we are unable to offer you the grant at this time. 
We encourage you to re-apply in the future, especially if you have any new, innovative ideas or approaches.

**Pull request closed without merging**